models/db/engine.go

```diff
@@ -123,7 +123,10 @@ func newXORMEngine() (*xorm.Engine, error) {
 
 // SyncAllTables sync the schemas of all tables, is required by unit test code
 func SyncAllTables() error {
-	return x.StoreEngine("InnoDB").Sync2(tables...)
+	_, err := x.StoreEngine("InnoDB").SyncWithOptions(xorm.SyncOptions{
+		WarnIfDatabaseColumnMissed: true,
```

Comment:
Should this not be false? Acording to it's option naming

Reply:
It should be true so that the warn will be shown, this is the only one place we need to show the warn because it will do an entire check.